cli/types/cypress.d.ts

```diff
@@ -281,6 +281,11 @@ declare namespace Cypress {
     specType?: CypressSpecType
   }
 
+  /**
+   * List of spec titlePaths (parent suite names + test name)
+   */
+  type TestFilter = string[] | null // could make this a callback/regex in the future
```

Comment:
Lets rename this to match the intent
```suggestion
  type CloudDebugTestList = string[] | null // could make this a callback/regex in the future
```